config/initializers/monkeypatch_vcr.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+if Rails.env.test? || Rails.env.development?
```

Comment:
@david-a-wheeler this isn't strictly necessary as you said but it shouldn't hurt

Reply:
Oh look, you already thought of that! Never mind!

Reply:
No just added it after you said that!